Shared/Logger.swift

```diff
@@ -19,7 +19,7 @@ public extension Logger {
     
     /// Logger used for things recorded on BraveRewards framework.
     static let rewardsLogger: RollingFileLogger = {
-        let logger = RollingFileLogger(filenameRoot: "rewards", logDirectoryPath: Logger.logFileDirectoryPath())
+        let logger = RollingFileLogger(filenameRoot: "rewards", logDirectoryPath: nil)
```

Comment:
There might be pre-existing logs. Can we delete the logs in the same change?

Reply:
this will be done with https://github.com/brave/brave-browser/issues/11260 from shared lib

Reply:
@jumde There will not be any pre-existing logs because this change (`nil` → `Logger.logFileDirectoryPath()`) was only introduced in the rewards internals PR which has not shipped to any live version yet

Reply:
@NejcZdovc Logging for iOS is different than Chromium, so any changes made to handle logs on brave-core side won't affect iOS. 

Also I thought that those changes would only ship with 1.13. This release is shipping with 1.12 lib so better to cut it off now

Reply:
yup this goes in 1.13, doesn't change anything for 1.12